starter-code/index.html

```diff
@@ -1,31 +1,152 @@
+
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <title>Instagram</title>
+ <link rel="stylesheet" href="./css/main.css">
+ <title>Instagram</title>
 </head>
 <body>
-  Instagram
 
-  Capture and Share the World's Moments
+// Blue background
+ <div class = "background">
+
+ <img class= "iphone" src="./images/phones.png" alt= "iphone">
 
-  Instagram is a fast, beautiful and fun way to share your life with friends and family.
+ <div class= "header">
+   <img class = "logo" src = "./images/brand.png" alt= "IG">
+   <button><img class = "log-in" src = "./images/home.png" alt= "Log In"><span>Log In</span></button>
+ </div>
+ </div>
 
-  Take a picture or video, choose a filter to transform its look and feel, then post to Instagram &mdash; it's that easy. You can even share to Facebook, Twitter, Tumblr and more. It's a new way to see the world.
+ <div class="white">
+   <img src="./images/frame.png" alt = "">
+ </div>
 
-  Oh yeah, did we mention it's free?
+// Information
+    <div class= "info">
+     <h3>Capture and Share the World's Moments</h3>
```

Comment:
this h3 heading should have had a bit more padding on top to drop it down a little